chalice/app.py

```diff
@@ -38,9 +38,14 @@ def handle_decimals(obj):
     return obj
 
 
-def error_response(message, error_code, http_status_code):
+def error_response(message, error_code, http_status_code, cors_headers=None):
     body = {'Code': error_code, 'Message': message}
     response = Response(body=body, status_code=http_status_code)
+    if cors_headers:
```

Comment:
No need to add the headers one at a time given that there aren't any other headers on the response at the moment. And in the future if they are we can rely on some up stream function to do the merging and just pass this one the final set of headers.

So how about changing `cors_headers` to just `headers` and passing it directly into the `Response` object.